playwright.config.ts

```diff
@@ -34,12 +34,12 @@ const config: PlaywrightTestConfig = {
     /* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
     actionTimeout: 0,
     /* Base URL to use in actions like `await page.goto('/')`. */
-    baseURL: `http://localhost:${process.env.PORT}`,
+    baseURL: `http://localhost:${process.env.PORT || 8080}`,
 
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-    trace: 'on-first-retry',
+    trace: 'retain-on-failure',
     /* Record video on first retry */
-    video: 'on-first-retry',
+    video: 'retain-on-failure',
```

Comment:
I don't think we need to change this, since we will get the same end result of a video for every failed test.  
This change will also make it record a video for every test and then discard videos for passed tests, which seems unnecessary. 
Is there a reason you changed this?

Reply:
I recall changing this back during our meeting, not sure why it reverted, I'll fix that now